hooks/src/index.js

```diff
@@ -170,11 +170,10 @@ export function useContext(context) {
 	const provider = currentComponent.context[context._id];
 	if (provider == null) return context._defaultValue;
 	const state = getHookState(currentIndex++);
-	if (state._value == null) {
-		state._value = true;
-		provider.sub(currentComponent);
-	}
-	return provider.props.value;
+	state.type = 1;
```

Comment:
Because I'm iterating over all hooks in `Provider` I need a way to find out if the hook at the current index is actually a `useContext` hook. We'll likely need something similar for the devtools soon.

Reply:
worth making each have a bit for this? `type` is a good property name since it's already elsewhere.

Reply:
Sounds good 👍

Reply:
I'd probably add the bit mask with #1383 . That change would be more appropriate there :)